egs/fisher_swbd/s5/local/chain/run_blstm_6h.sh

```diff
@@ -16,8 +16,12 @@ decode_dir_affix=
 num_epochs=4
 remove_egs=false
 common_egs_dir=
-
 affix=
+
+# decoding options
+extra_left_context=50
```

Comment:
other than this change every thing else looks good to me. I will merge after this change is done.